p/scripts/main.js

```diff
@@ -857,6 +857,13 @@ function init_shortcuts() {
 				return false;
 			}
 		}
+		if (k === s.actualize) {
+			const btn = document.getElementById('actualize');
+			if (btn) {
+				btn.click();
+			}
```

Comment:
Why not just call the action directly?

Reply:
I see. I suppose that's fine then, but it might mean it's time to refactor it out. I'll defer to @Alkarex.
https://github.com/FreshRSS/FreshRSS/blob/9326e419e3b90b2db4b6a8531503b710605ac6d4/p/scripts/main.js#L1234-L1286

Reply:
I might have missed a comment. What is your suggestion, @Frenzie ?

Reply:
Conceptually I'd expect these to be separate actions that trigger the same function, rather than that other actions trigger a click event on the button that triggers the function.